src/sdk/actions.ts

```diff
@@ -130,3 +142,47 @@ async function returnDataToKernel(repoToken: string, stateId: string, output: ob
     },
   });
 }
+
+function createStructuredMetadata(className: string | undefined, logReturn: LogReturn) {
```

Comment:
I know I used this function before but I don't remember what `className` is supposed to be?

Reply:
It's developer defined and can be set as anything for a unique identifier. The class of the log. If this turns out to be unreliable then we can target the function name instead. I designed the metadata header for flexibility because I don't know what will be adopted and the most reliable string to target.